### ISSUE TITLE
Docs app not working with Dash 2.14.0

### ISSUE THREAD

**Issue by AnnMarieW** [facultyai/dash-bootstrap-components#983]
It looks like the docs app doesn't work with  the latest release of Dash 2.14.0.  It works fine in 2.13.0

I think it has something to do with this change:  [github.com/plotly/dash](https://github.com/plotly/dash/commit/a83c5b3087a2b74de2a0b3cf7b1a67260ae5f97f)

Here’s the error
```
/home/amw/PycharmProjects/dash-bootstrap-components/venv/bin/python /home/amw/PycharmProjects/dash-bootstrap-components/docs/run.py 
Traceback (most recent call last):
  File "/home/amw/PycharmProjects/dash-bootstrap-components/docs/run.py", line 13, in <module>
    example_routes = register_example_apps()
  File "/home/amw/PycharmProjects/dash-bootstrap-components/docs/examples/__init__.py", line 109, in register_apps
    exec(
  File "<string>", line 17, in <module>
  File "/home/amw/PycharmProjects/dash-bootstrap-components/venv/lib/python3.10/site-packages/dash/dash.py", line 399, in __init__
    caller_name = get_caller_name(self.__class__.__name__)
  File "/home/amw/PycharmProjects/dash-bootstrap-components/venv/lib/python3.10/site-packages/dash/_utils.py", line 290, in get_caller_name
    for code in s.code_context:
TypeError: 'NoneType' object is not iterable
```

**Comment by AnnMarieW** [comment]
Looks like a patch is in the works:

https://community.plotly.com/t/dash-object-is-not-loading/79596/3?u=annmariew

**Comment by AnnMarieW** [comment]
This is now fixed in dash 2.14.1.  So the docs app now works with every version of dash except 2.14.0

**Comment by tcbegley** [comment]
Great, thanks for the update! I think we can close this then with no action required.
